ui/components/app/smart-transactions-modal/smart-transactions-modal.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  TextColor,
+  Display,
+  FlexDirection,
+  FontWeight,
+  BlockSize,
+  AlignItems,
+  JustifyContent,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  Modal,
+  ModalOverlay,
+  Text,
+  Box,
+  Button,
+  ButtonVariant,
+  ModalHeader,
+  ModalContent,
+} from '../../../components/component-library';
+
+interface Props {
+  onContinue: () => void;
+  onNoRightNow: () => void;
+  isOpen: boolean;
+}
+
+export default function SmartTransactionsModal({
+  onContinue,
+  onNoRightNow,
+  isOpen,
+}: Props) {
+  const t = useI18nContext();
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={onContinue}
+      isClosedOnOutsideClick={false}
+      isClosedOnEscapeKey={false}
+      className="mm-modal__custom-scrollbar"
+      autoFocus={false}
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader
+          alignItems={AlignItems.center}
+          justifyContent={JustifyContent.center}
+        >
+          {t('introducingSmartTransctions')}
+        </ModalHeader>
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Column}
+          paddingLeft={4}
+          paddingRight={4}
+        >
+          <Box display={Display.Flex} flexDirection={FlexDirection.Column}>
+            <img
```

Comment:
Which values would you use for width / height? We want to stretch it based on the size of the modal, because the full-size view and popup view have different width.
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/1366f5f2-1bdb-43fe-a5a3-ec995c0c87e6)